editor/filesystem_dock.cpp

```diff
@@ -1004,6 +1004,9 @@ void FileSystemDock::_update_file_list(bool p_keep_selection) {
 			}
 		}
 	} else {
+		if (!directory.begins_with("res://")) {
+			directory = "res://" + directory;
+		}
```

Comment:
An error is printed.

Reply:
Yes it will print something like "Cannot navigate there"

I was thinking whether we should make this field uneditable like @KoBeWi suggested in the issue, not sure if anyone is relying on it.